examples/workflow/4_analyze_results.ipynb

```diff
@@ -0,0 +1,415 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Applying a transformation to one of the columns is also easily done for a `pandas.DataFrame`. For instance, if you are interested in the invariant mass (and not the **square value** of the invariant mass), simply do:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from numpy import sqrt\n",
+    "\n",
+    "frame_data['M(3,4)'] = sqrt(frame_data['mSq_(3,4)'])\n",
+    "frame_phsp['M(3,4)'] = sqrt(frame_phsp['mSq_(3,4)'])\n",
+    "\n",
+    "plot_1d_comparison('M(3,4)')\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dalitz plots"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Dalitz plots are 2-dimensional histograms of the square values of the invariant masses. We are therefore interested in the following variables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "variable_names = data_set.data.keys()\n",
+    "[var for var in variable_names if var.startswith('mSq_')]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now use [matplotlib.pyplot.hist2d](https://matplotlib.org/api/_as_gen/matplotlib.pyplot.hist2d.html) to generate the Dalitz plots:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def dalitz_plot(frame, mass_x, mass_y, bins=50, **kwargs):\n",
+    "    \"\"\"Helper function to create a Dalitz plot with useful axis titles.\"\"\"\n",
+    "    plt.hist2d(\n",
+    "        frame[mass_x],\n",
+    "        frame[mass_y],\n",
+    "        weights=frame['weights'],\n",
+    "        bins=bins,\n",
+    "        **kwargs,\n",
+    "    )\n",
+    "    plt.xlabel(replace_ids(mass_x))\n",
+    "    plt.ylabel(replace_ids(mass_y))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from matplotlib.colors import LogNorm  # logarithmic z-axis\n",
+    "\n",
+    "fig, axs = plt.subplots(1, 2, figsize=(9, 4))\n",
+    "\n",
+    "plt.sca(axs[0])\n",
+    "axs[0].set_title('data')\n",
+    "dalitz_plot(frame_data, 'mSq_(3,4)', 'mSq_(2,4)', norm=LogNorm())\n",
+    "\n",
+    "plt.sca(axs[1])\n",
+    "axs[1].set_title('fit')\n",
+    "dalitz_plot(frame_phsp, 'mSq_(3,4)', 'mSq_(2,4)', norm=LogNorm())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3 Calculate fit fractions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The original intensity model that we optimized consists of several components. All registered component names can be retrieved via ``get_all_component_names()``:"
```

Comment:
It might also be better to change that to raw to be able to link it to the method

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/51b48a8618afb2997e753951df616771232a213c